packaging/commands/commander.go

```diff
@@ -132,9 +133,21 @@ func (p *packageCommander) proxyConfigLines(settings proxy.Settings) []string {
 		}
 	}
 
-	addOption("http", settings.Http)
-	addOption("https", settings.Https)
-	addOption("ftp", settings.Ftp)
+	// OpenSUSE uses proxy labels in capital letter (e.g HTTP_PROXY)
+	// For backward compatibility I included a flag in packageCommander.
+	var http_label, https_label, ftp_label string
+	if p.proxyLabelInCapital{
+		http_label = "HTTP"
+		https_label = "HTTPS"
+		ftp_label = "FTP"
+	}else{
```

Comment:
There's some non-standard formatting in this file, please run "go fmt" over all files. (I would do it myself after, but I think the merge bot will reject the merge if the files aren't formatted.)